### PR TITLE
fix shadowing bug

### PR DESCRIPTION
**Pull request by lavalamp**
#21522 gave a useless message due to this.

### CODE REVIEW

File: test/e2e/e2e.go [hunk 1]
```diff
@@ -182,9 +182,11 @@ var _ = ginkgo.SynchronizedBeforeSuite(func() []byte {
 	// test pods from running, and tests that ensure all pods are running and
 	// ready will fail).
 	if err := waitForPodsRunningReady(api.NamespaceSystem, testContext.MinStartupPods, podStartupTimeout); err != nil {
-		c, err := loadClient()
-		Expect(err).NotTo(HaveOccurred())
-		dumpAllNamespaceInfo(c, api.NamespaceSystem)
+		if c, errClient := loadClient(); errClient != nil {
+			Expect(errClient).NotTo(HaveOccurred())
```
Comment (ixdy) [inline test/e2e/e2e.go]: probably don't need to if-guard this - `Expect` will bail out if it fails.

Comment (lavalamp) [inline test/e2e/e2e.go]: Oh. I don't want that, then we never see the error. I'll make a change.

**Comment by lavalamp** [comment]
Fixed. PTAL

File: test/e2e/e2e.go [hunk 2]
```diff
@@ -182,9 +181,11 @@ var _ = ginkgo.SynchronizedBeforeSuite(func() []byte {
 	// test pods from running, and tests that ensure all pods are running and
 	// ready will fail).
 	if err := waitForPodsRunningReady(api.NamespaceSystem, testContext.MinStartupPods, podStartupTimeout); err != nil {
-		c, err := loadClient()
-		Expect(err).NotTo(HaveOccurred())
-		dumpAllNamespaceInfo(c, api.NamespaceSystem)
+		if c, errClient := loadClient(); errClient != nil {
+			Logf("Unable to dump cluster information because: %v", err)
```
Comment (ixdy) [inline test/e2e/e2e.go]: should this be `errClient`?

**Comment by lavalamp** [comment]
Flake is #21451

**Comment by lavalamp** [comment]
Fixed comment, rebased. PTAL

**Comment by ixdy** [comment]
LGTM

**Comment by lavalamp** [comment]
@k8s-oncall this is a flake debugging helper, please manually merge

**Pull request merged**